### ISSUE TITLE
Empty survex file

### ISSUE THREAD

**Issue by Ospeleo** [detrito/toporobot2survex#1]
on MacOs 10.12, I compiled the files (make)

then launched ` ./toporobot2survex toporobotfile_01 .`

The program print
```
Processing input file...
1 
2 
3 
4 
5 
6 
7 
8 
9 
10 
11 
12 
13 
14 
15 
16 
17 
18 
19 
Writing to output files...
Writing cave... 
Closing cave...
Exiting.
```

However, the outpout survex file is valid, but empty : 

```
*begin
*title ""

*equate entree s1.0
*fix entree 0 0 0



*end
```
It does the same with other toporobot files that are known to work.

Is there a known problem with toporobot2survex ? some data that pro

**Comment by detrito** [comment]
Ospeleo <notifications@github.com> wrote:

> then launched ` ./toporobot2survex toporobotfile_01 .`

hi, can I have a look at your "toporobotfile_01" input file in order to
understand what's going on?

cheers, d

-- 
gpg --recv-key 0x88D6D3C6

**Comment by Ospeleo** [comment]
thanks,
I found an old PC, installed GHTopo (Win only), and GHTopo was able to open the .Text and convert.
However, It's not very practical....  I have some old surveys, so maybe a hundred file to convert
You can find one .Text fileIt would be interesting to know why it doesn't work.
thanks for the software, it is to my knowledge the only one that can do this job
Olivier Testa
 

    Le mercredi 12 décembre 2018 à 23:36:31 UTC+1, detrito <notifications@github.com> a écrit :  
 
 Ospeleo <notifications@github.com> wrote:

> then launched ` ./toporobot2survex toporobotfile_01 .`

hi, can I have a look at your "toporobotfile_01" input file in order to
understand what's going on?

cheers, d

-- 
gpg --recv-key 0x88D6D3C6

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
   
    -6     1   1   1   1 Sortie Sene
    -6     2   1   1   1 Thalma
    -6     3   1   1   1 
    -5     1   1   1   1   788035.00  2030112.00     1911.00     1     7
    -5     2   1   1   1   788028.00  2030727.00     2010.00     5    11
    -5     3   1   1   1   787719.00  2030925.00     2068.00     6     0
    -4     1   1   1   1 13/02/27 17:35:53  Guest
    -4     2   1   1   1 13/02/27 17:35:53  Guest
    -3     1   1   1   1
    -2     1   1   1   1 01011900                              0    0.00   0   1
    -2     2   1   1   1 14/02/13  O Testa / S×  O Testa       0   10.81   0   1
(   -2     2   1   1   1-Jagouü1
    -2     3   1   1   1 13/02/13  O Testa - JF  O Testa       0   10.80   0   1
(   -2     3   1   1   1- Fabriolü1
    -1     1   1   1   1  360.00  360.00    0.10    1.00    1.00  100.00    0.00
     1    -2   1   1   1
     1    -1   1   1   1       1       0       1       7       7       3       0
     1     0   1   1   1    0.00    0.00    0.00    0.00    0.00    0.00    0.00
     1     1   1   1   1  371.00  219.00  -13.00    0.00    0.00    0.00    0.00
     1     2   1   1   1   65.00  168.00    0.00    0.00    0.00    0.00    0.00
     1     3   1   1   1   33.00  153.00    0.00    0.00    0.00    0.00    0.00
     1     4   1   1   1   16.00  116.00    0.00    0.00    0.00    0.00    0.00
     1     5   1   1   1   63.00  213.00  -10.00    0.00    0.00    0.00    0.00
     1     6   1   1   1   22.00  267.00    0.00    0.00    0.00    0.00    0.00
     1     7   1   1   1   58.00  218.00  -20.00    0.00    0.00    0.00    0.00
     2    -2   1   1   1
     2    -1   1   1   1       1       0       2       5       5       3       0
     2     0   1   1   1    0.00    0.00    0.00    0.00    0.00    0.00    0.00
     2     1   1   1   1   22.00  122.00    0.00    0.00    0.00    0.00    0.00
     2     2   1   1   1   41.00  234.00  -13.00    0.00    0.00    0.00    0.00
     2     3   1   1   1  145.00  218.00  -13.00    0.00    0.00    0.00    0.00
     2     4   1   1   1  101.00  201.00  -13.00    0.00    0.00    0.00    0.00
     2     5   1   1   1   98.00  240.00  -13.00    0.00    0.00    0.00    0.00
     3    -2   1   1   1
     3    -1   1   1   1       1       2       3       2       2       3       0
     3     0   1   1   1    0.00    0.00    0.00    0.00    0.00    0.00    0.00
     3     1   1   1   1  153.00  336.00    0.00    0.00    0.00    0.00    0.00
     3     2   1   1   1  236.00   61.00    0.00    0.00    0.00    0.00    0.00
     4    -2   1   1   1
     4    -1   1   1   1       4       0       4       8       8       3       0
     4     0   1   1   2    0.00    0.00    0.00    1.70    2.40    2.40    2.60
     4     1   1   1   2   73.88   33.00   14.00    2.70    2.30    7.70    1.70
     4     2   1   1   2    9.50   39.00   23.00    1.90    1.15    6.10    1.70
     4     3   1   1   2   12.69   71.00   22.00    1.10    0.00    4.10    1.10
     4     4   1   1   2    6.90    2.00   30.00    0.20    1.30    3.50    1.60
     4     5   1   1   2   13.43   53.00   10.00    0.20    1.80    5.80    1.70
     4     6   1   1   2   10.13  117.00   22.00    1.00    0.70    1.50    1.80
     4     7   1   1   2    4.72   53.00    8.00    0.50    0.50    3.00    2.00
     4     8   1   1   2   10.57   10.00    0.00    1.40    2.00    5.80    1.40
     5    -2   1   1   1
     5    -1   1   1   1       4       8       5      11      11       3       0
     5     0   1   1   2    0.00    0.00    0.00    0.00    2.00    5.80    1.40
     5     1   1   1   2    6.57  129.00   18.00    2.00    0.20    3.80    1.70
     5     2   1   1   2   11.61   93.00    8.00    0.20    1.30    3.50    1.70
     5     3   1   1   2    2.67  162.00   16.00    0.40    0.20    2.30    1.00
     5     4   1   1   2    7.48  110.00    0.00    0.20    0.70    1.80    0.60
     5     5   1   1   2    5.23   90.00    0.00    0.20    0.80    2.00    0.80
     5     6   1   1   2    4.37  130.00    7.00    0.40    0.40    1.40    0.60
     5     7   1   1   2   28.80  108.00    1.00    4.00    0.70    2.00    0.50
     5     8   1   1   2   10.41   71.00   12.00    5.00    4.00    4.00    1.70
     5     9   1   1   2   37.76   26.00   21.00    0.80    0.80    2.30    1.70
     5    10   1   1   2    4.08  305.00   12.00    2.18    2.00    4.80    1.70
     5    11   1   1   2   10.00   20.00    8.00    8.70    3.00    4.80    1.70
     6    -2   1   1   1
     6    -1   1   1   1       6       0       6      13      13       3       0
     6     0   1   1   3    0.00    0.00    0.00    4.00   10.00   12.00    1.60
     6     1   1   1   3   43.65  221.00  -26.00    3.30    2.10    9.70    1.60
     6     2   1   1   3   10.26  233.00  -23.00    7.30    3.30   14.50    1.50
     6     3   1   1   3   13.16  276.00   -7.00    2.70    2.10   14.00    2.00
     6     4   1   1   3   16.77  216.00  -16.00    2.80    4.70    8.70    8.30
     6     5   1   1   3   34.95  252.00  -14.00    6.20    2.30    6.50    4.30
     6     6   1   1   3   20.77  196.00  -25.00    1.80    6.60    7.20    1.40
     6     7   1   1   3   20.06  268.00  -10.00    4.20    1.70    6.00    2.30
     6     8   1   1   3   14.22  206.00  -10.00    5.10    2.60    5.00    2.20
     6     9   1   1   3   13.26  253.00  -32.00    7.80    1.80    8.80    1.60
     6    10   1   1   3   36.16  191.00  -22.00    3.20    4.10   10.10    2.30
     6    11   1   1   3   50.23  205.00  -10.00    6.50    4.15    3.30    1.50
     6    12   1   1   3   18.88  193.00  -18.00    5.80   12.60    2.10    1.60
     6    13   1   1   3   12.53  230.00  -42.00    3.50   14.30    3.20    0.60
     7    -2   1   1   1
     7    -1   1   1   1       6      13       7      10      10       3       0
     7     0   1   1   3    0.00    0.00    0.00    3.50    6.90    3.20    3.60
     7     1   1   1   3   31.11  188.00  -13.00   17.10    5.20    8.10    3.00
     7     2   1   1   3   10.28  218.00  -16.00    1.00    1.00    7.40    1.70
     7     3   1   1   3   10.58  166.00  -14.00    5.20    3.70    1.70    1.70
     7     4   1   1   3   24.80  194.00  -32.00    0.00    4.00    6.00    1.70
     7     5   1   1   3   15.37  215.00  -17.00    3.60    1.00    6.60    2.00
     7     6   1   1   3   12.55  163.00  -19.00    1.90    2.60    6.00    1.60
     7     7   1   1   3    9.51  220.00  -28.00    1.60    1.00    6.90    2.20
     7     8   1   1   3    6.29  147.00   -7.00    1.60    3.00    6.20    2.20
     7     9   1   1   3    3.05  207.00  -19.00    1.60    1.00    5.20    1.30
     7    10   1   1   3    3.40  188.00  -68.00    0.00    0.00    0.00    0.00
     8    -2   1   1   1
     8    -1   1   1   1       7      10       8      28      28       3       0
     8     0   1   1   3    0.00    0.00    0.00    0.50    4.00    0.00    1.50
     8     1   1   1   3    5.65  256.00  -22.00    1.60    0.50    0.50    0.50
     8     2   1   1   3    5.56  228.00   -3.00    1.00    1.00    0.30    0.50
     8     3   1   1   3    4.07  237.00   35.00    3.10    3.30    4.70    2.80
     8     4   1   1   3   19.30  205.00   -9.00    2.40    4.30    4.80    4.30
     8     5   1   1   3   11.55  208.00  -22.00    4.40    0.50    1.80    1.70
     8     6   1   1   3   19.76  160.00  -11.00    2.70    4.00    4.40    2.80
     8     7   1   1   3   10.62  163.00  -28.00    2.30    0.80    4.30    1.70
     8     8   1   1   3    8.82  124.00  -10.00    0.30    2.30    1.00    1.80
     8     9   1   1   3   11.18  178.00   -5.00    2.50    3.70    4.50    1.70
     8    10   1   1   3   11.40  163.00   -7.00    3.00    2.30    5.40    2.00
     8    11   1   1   3   17.96  164.00  -15.00    0.50    6.00    5.50    2.90
     8    12   1   1   3   21.96  229.00  -11.00    5.90    0.50    0.50    1.70
     8    13   1   1   3   25.06  205.00  -30.00    0.50    3.60    5.30    1.80
     8    14   1   1   3    6.30  245.00    6.00    0.50    3.60    2.40    1.70
     8    15   1   1   3   10.38  208.00  -46.00    4.00    1.20    1.80    6.10
     8    16   1   1   3   13.48  191.00  -38.00    0.70    3.90    4.20    1.70
     8    17   1   1   3   17.50  234.00  -16.00    1.20    0.60   12.20    2.30
     8    18   1   1   3   30.72  214.00   -5.00    2.80    1.20    6.50    2.60
     8    19   1   1   3   14.70  215.00   -6.00    1.40    4.20    4.60    1.90
     8    20   1   1   3   12.40  245.00  -35.00    1.50    0.50    6.00    2.60
     8    21   1   1   3   28.78  212.00  -11.00    1.00    1.40    5.60    6.00
     8    22   1   1   3   19.90  214.00  -18.00    2.00    0.40    6.70    1.80
     8    23   1   1   3    4.15  182.00   -8.00    0.50    0.50    7.00    2.00
     8    24   1   1   3   22.85  215.00  -21.00    1.80    0.20    5.50    2.00
     8    25   1   1   3   20.70  210.00   -8.00    4.00    0.40   21.00    2.00
     8    26   1   1   3   24.33  207.00  -11.00    2.18    1.40    3.20    2.50
     8    27   1   1   3   20.81  218.00  -20.00    0.30    0.30    2.00    1.70
     8    28   1   1   3    5.98  136.00   -4.00    0.30    0.30    2.00    1.70
     9    -2   1   1   1
     9    -1   1   1   1       8      28       9       5       5       3       0
     9     0   1   1   3    0.00    0.00    0.00    3.70    1.80    1.70    1.70
     9     1   1   1   3   13.20  201.00  -12.00    1.40    1.00    0.50    2.50
     9     2   1   1   3    6.33  216.00   -8.00    0.60    0.30    0.50    2.50
     9     3   1   1   3    4.40  157.00  -12.00    0.30    1.50    0.60    1.40
     9     4   1   1   3    6.18  217.00  -20.00    0.40    0.40    1.60    1.80
     9     5   1   1   3    5.78  220.00   -3.00    0.40    0.40    1.60    1.80
    10    -2   1   1   1
    10    -1   1   1   1       9       5      10       1       1       3       0
    10     0   1   1   3    0.00    0.00    0.00    2.60    0.60    3.40    1.70
    10     1   1   1   3    6.77  158.00   -4.00    4.50    0.00    0.40    4.00
    11    -2   1   1   1
    11    -1   1   1   1      10       1      11       3       3       3       0
    11     0   1   1   3    0.00    0.00    0.00    2.60    2.80    0.40    4.00
    11     1   1   1   3   12.01  221.00  -14.00    0.40    1.20    0.50    4.00
    11     2   1   1   3   15.60  196.00  -15.00    0.20    2.00    2.20    2.50
    11     3   1   1   3   10.31  230.00   -2.00    0.50    1.50    0.50    1.50

**Comment by detrito** [comment]
Ospeleo <notifications@github.com> wrote:

>     -6     1   1   1   1 Sortie Sene
>     -6     2   1   1   1 Thalma
>     -6     3   1   1   1 
>     -5     1   1   1   1   788035.00  2030112.00     1911.00
> 1     7 -5     2   1   1   1   788028.00  2030727.00     2010.00

your file looks like a toporobot "text" format (format for native data)
and not a toporobot tab format (format for export data). but maybe the
formatting of the file got just lost in your mail.

the official doc about all this went offline but was originally here:
http://www.geo.uzh.ch/~heller/toporobot/Francais/Fournisseurs/FormatsToporobot/Text.html

if you e-mail me your raw file I could check it this we.

cheers, d.

-- 
gpg --recv-key 0x88D6D3C6

**Comment by Ospeleo** [comment]
thanks,I don't know why it plain text on your side. It's properly tabulated on my mac. Maybe character encoding ?
I'll look into that
Goodnight.where are you ? I'm in France
Olivier Testa


 

    Le jeudi 13 décembre 2018 à 00:05:36 UTC+1, detrito <notifications@github.com> a écrit :  
 
 Ospeleo <notifications@github.com> wrote:

> -6 1 1 1 1 Sortie Sene
> -6 2 1 1 1 Thalma
> -6 3 1 1 1 
> -5 1 1 1 1 788035.00 2030112.00 1911.00
> 1 7 -5 2 1 1 1 788028.00 2030727.00 2010.00

your file looks like a toporobot "text" format (format for native data)
and not a toporobot tab format (format for export data). but maybe the
formatting of the file got just lost in your mail.

the official doc about all this went offline but was originally here:
http://www.geo.uzh.ch/~heller/toporobot/Francais/Fournisseurs/FormatsToporobot/Text.html

if you e-mail me your raw file I could check it this we.

cheers, d.

-- 
gpg --recv-key 0x88D6D3C6

**Comment by detrito** [comment]
Ospeleo <notifications@github.com> wrote:

> thanks,I don't know why it plain text on your side.

maybe because of the github UI. please send me the raw file at
detrito{_at-]inventati.org, I will have a look at it ;)

-- 
gpg --recv-key 0x88D6D3C6

**Comment by detrito** [comment]
Should be fixed in commit c29c606
